src/Common/src/Interop/Unix/System.Native/Interop.Stat.cs

```diff
@@ -8,6 +8,7 @@ internal static partial class Interop
 {
     internal static partial class Sys
     {
+        [StructLayout(LayoutKind.Sequential)]
```

Comment:
Can you add a comment for why this is being specified explicitly even though it's the default?